### ISSUE TITLE
CCU firmware download fails

### ISSUE THREAD

**Issue by jtreml** [angelnu/docker-ccu#39]
The URL for the firmware download seems to be wrong, causing the firmware download during setup to fail.

```
Step 5/23 : RUN export CCU_FW_LINK="http://update.homematic.com/firmware/download?cmd=download&version=${CCU_VERSION}&serial=${CCU_SERIAL}&lang=de&product=HM-CCU${CCU_VERSION%%.*}"     && echo "Downloading from $CCU_FW_LINK "     && wget --no-verbose $CCU_FW_LINK -O -|tar -xzO rootfs.ext4.gz|gunzip>rootfs.ext4
 ---> Running in 7c332aee9f75
Downloading from http://update.homematic.com/firmware/download?cmd=download&version=3.47.22&serial=ccu_docker&lang=de&product=HM-CCU3
http://update.homematic.com/firmware/download?cmd=download&version=3.47.22&serial=ccu_docker&lang=de&product=HM-CCU3:
2020-02-19 08:29:44 ERROR 400: Bad Request.

gzip: stdin: unexpected end of file
tar: Child returned status 1
tar: Error is not recoverable: exiting now

gzip: stdin: unexpected end of file
```

**Comment by angelnu** [comment]
The download link changed... I will update the build to use the second newest link which is not protected by a cookie. Could you please use this and then, when the UI suggest you to download the new firmware, copy here the link it offers to download it?

**Comment by angelnu** [comment]
Thanks to the suggestion in #38 I fixed the download - Updated to 3.51.6
